challenge.rb

```diff
@@ -67,6 +89,44 @@ def ac_monitor current_temp, ac_working, desired_temp
 # and print out the result of `fizz_buzz_calculator` for that number.
 # This question will be graded independently of question four.
 
-# Replace this comment with your code.
 
+def fizz_buzz(min, max)
+    (min..max).each { |num| puts fizz_buzz_calculator(num) }
+end
+
+
+
+
+# Testing
```

Comment:
Whoa! Nice job! Testing your code is an excellent practice to follow. At some point in this unit we'll cover a more sophisticated tool for writing tests in Ruby called Rspec - feel free to take a look at it!

http://rspec.info/